common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_bml.yaml

```diff
@@ -20,6 +20,16 @@ cACTION_BML_PING_RESPONSE:
 
 cACTION_BML_NW_MAP_REQUEST:
   _type: class
+  
+cACTION_BML_OPERATIONAL_STATUS_RESPONSE:
+  _type: class
+  buffer_size:
+    _type: uint32_t
+    _length_var: True
+  buffer: 
```

Comment:
Why buffer and not actual response parameters?

Reply:
done